torch_points3d/data/batch.py

```diff
@@ -0,0 +1,58 @@
+import torch
+from torch_geometric.data import Data
+
+
+class SimpleBatch(Data):
```

Comment:
What's the difference between SimpleBatch and normal Data object?

Reply:
The difference is that SimpleBatch is used for  dense convolutions (the size of point clouds are always the same). so the tensors have the following shape `[B, P, F]`.  It is used for PointNet++ and RSConv (It is Faster like that). for Batch it has the size `[P, F]` with an attribute called batch of size `[P]`.